src/entryPoints/SidebarValues.res

```diff
@@ -466,15 +466,93 @@ let developers = (isDevelopersEnabled, userRole, systemMetrics, ~permissionJson)
     : emptyComponent
 }
 
-let reconTag = (recon, isReconEnabled) =>
-  recon
-    ? Link({
-        name: "Reconcilation",
-        icon: isReconEnabled ? "recon" : "recon-lock",
-        link: `/recon`,
-        access: Access,
-      })
-    : emptyComponent
+let uploadReconFiles = () => {
+  SubLevelLink({
+    name: "Upload Recon Files",
+    link: `/upload-files`,
+    access: Access,
+    searchOptions: [("Upload recon files", "")],
+  })
+}
+
+let runRecon = () => {
+  SubLevelLink({
+    name: "Run Recon",
+    link: `/run-recon`,
+    access: Access,
+    searchOptions: [("Run recon", "")],
+  })
+}
+
+let reconAnalytics = () => {
+  SubLevelLink({
+    name: "Analytics",
+    link: `/recon-analytics`,
+    access: Access,
+    searchOptions: [("Recon analytics", "")],
+  })
+}
+let reconReports = () => {
+  SubLevelLink({
+    name: "Reports",
+    link: `reports`,
+    access: Access,
+    searchOptions: [("Recon reports", "")],
+  })
+}
+
+let reconConfigurator = () => {
+  SubLevelLink({
+    name: "Configurator",
+    link: `config-settings`,
+    access: Access,
+    searchOptions: [("Recon configurator", "")],
+  })
+}
+let reconFileProcessor = () => {
+  SubLevelLink({
+    name: "File Processor",
+    link: `file-processor`,
+    access: Access,
+    searchOptions: [("Recon file processor", "")],
+  })
+}
+
+let reconTag = (recon, isReconEnabled) => {
+  let uploadReconFiles = uploadReconFiles()
+  let runRecon = runRecon()
+  let reconAnalytics = reconAnalytics()
+  let reconReports = reconReports()
+  let reconConfigurator = reconConfigurator()
+  let reconFileProcessor = reconFileProcessor()
```

Comment:
can be moved to inside the ` | (true, true, true) =>` case as otherwise they are declared but never used